.editorconfig

```diff
@@ -1,9 +1,9 @@
 root = true
 
 [*]
-indent_style = space
+indent_style = tab
 charset = utf-8
 end_of_line = lf
-indent_size = 2
+indent_size = 4
```

Comment:
Not happy with the tab design decision. I believe the majority of them using 2 line spacing currently. Use `--space`